packages/autoloader/src/AutoloadGenerator.php

```diff
@@ -0,0 +1,311 @@
+<?php
+// phpcs:ignoreFile -- this is not a core file
```

Comment:
Do we need this? It seems that a lot of the warnings are about comments (that can be fixed) or `WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase` (we could add an exception for this rule only for the whole file).

Reply:
I update the packages php files to only ignore rules that I though I can't fix because they really require the standards of composer to be implemented since these packages assume that the dev is running a latter version of PHP on the dev machine